test.js

```diff
@@ -72,6 +72,29 @@ describe('set', function () {
   it('should set at a numeric path', function () {
     expect(op.set([], 0, 'yo')).to.deep.equal(['yo'])
   })
+
```

Comment:
Could you also add a test for the new method in the following sections, just to be sure ti works with the bind mode as well?

* should execute all methods on the bound object
* should throw if an operation is attempted after `value` called